dashboard.py

```diff
@@ -526,34 +540,109 @@ def make_ab_html_report(
     doc.save(
         out_fname,
         title="A/B confidence intervals vs. " + baseline,
-        resources=INLINE,
+        resources=CDN,
     )
     return True
 
 
+def make_details_html_report(
+    df: pandas.DataFrame,
+    output_dir: pathlib.Path,
+    runtime: str,
+    fullname: str,
+) -> None:
+    """Generate raw tabular info dump for all runs of a single test"""
+    df = df.reset_index()
+    # Delete redundant columns
+    for k in (
+        "name",
+        "originalname",
+        "path",
+        "coiled_runtime_version",
+        "coiled_software_name",
+        "python_version",
+        "category",
+        "sourcename",
+        "runtime",
+        "fullname",
+        "fullname_no_category",
+    ):
+        del df[k]
+
+    header = list(df.columns) + ["grafana_url"]
+    txt = dedent(
+        f"""
+        <style>
+            table, th, td {{border: 1px solid black; border-collapse: collapse;}}
+            th {{position: sticky; top: 0; background-color: lightgrey;}}
+        </style>
+        ### {runtime}
+        ### {fullname}
+        | {' | '.join(header)} |
+        | {' | '.join("---" for _ in header)} |
+        """
+    )
+
+    for id_, ds_row in df.iterrows():
+        row = ds_row.to_dict()
+
+        if cluster_name := row["cluster_name"]:
+            # Add some padding to compensate for clock differences between
+            # GitHub actions and Prometheus (we've observed 80s gaps)
+            ts_padding = pandas.Timedelta("5m")
+            start_ts = int((row["start"] - ts_padding).timestamp() * 1000)
+            end_ts = int((row["end"] + ts_padding).timestamp() * 1000)
+            row["grafana_url"] = (
+                "http://35.86.202.18:3000/d/eU1bT-nVz/cluster-metrics-prometheus"
```

Comment:
Let's use the new URL I just setup:

```suggestion
                "https://grafana.dev-sandbox.coiledhq.com/d/eU1bT-nVz/cluster-metrics-prometheus"
```